### DISCUSSION TITLE
Upload working directory structure in a new branch?

### DISCUSSION THREAD

**Discussion by Unloaded123456** [Abstrat-Technologies/rustedanvil#18] [Technical]
If Forge #965 can run, there should be a directory structure, with this, people would put the necessary files and make the stuff work! {it does not matter how much time it would take, if you have a working MDK installation, uploading directory structure in the early stages of the project would be good}.

**Comment by arek-kwapis** [comment]
This has been addressed in the issue you opened for it (#19) - It won't be done due to time constraints on my part.

We're almost at Milestone 2 anyway, so it doesn't matter too much
